### PR TITLE
[stable/kong] fix link to Custom Resource Definitions  in README.md 

### PR DESCRIPTION
**Pull request by dkozlov**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/kong/README.md [hunk 1]
```diff
@@ -363,7 +363,7 @@ The custom resources are:
 
 You can can learn about kong ingress custom resource definitions here:
```
Comment (hbagdi) [inline stable/kong/README.md]: Can we instead hyperlink with the text "here"?

Meaning, make "here" link to the link below?

**Review by hbagdi — CHANGES_REQUESTED** [review]

File: stable/kong/README.md (hunk 1 above)
Comment (dkozlov) [reply stable/kong/README.md]: done in https://github.com/helm/charts/pull/14606/commits/c4d3bd39928d09cd5b71bde89fc0eefbd0b2d5c6

**Review by hbagdi — APPROVED** [review]
/ok-to-test
/lgtm

**Pull request merged**